### PR TITLE
Update ZHANGTIANYAO1 AboutUS

### LINKED ISSUE #1: Update AboutUs
Each person is responsible for updating their own information.

### PR DESCRIPTION
**Pull request by ZHANGTIANYAO1**
Update ZHANGTIANYAO1 AboutUS

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Zhang Tianyao
```
Comment (dabzpengu) [inline docs/AboutUs.md:12]: looks good to me

File: docs/AboutUs.md [hunk 2]
```diff
@@ -9,13 +9,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+<img src="images/zhangtianyao1.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[homepage](https://github.com/ZHANGTIANYAO1)]
+[[github](https://github.com/ZHANGTIANYAO1)]
+[[portfolio](team/zhangtianyao.md)]
 
 * Role: Project Advisor
```
Comment (dabzpengu) [inline docs/AboutUs.md:20]: looks good to me, fitting role!

**Review by dabzpengu — APPROVED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (Lalelulilulela) [reply docs/AboutUs.md:12]: Fabulous name

File: docs/AboutUs.md (hunk 2 above)
Comment (Lalelulilulela) [reply docs/AboutUs.md:20]: Fabulous role

**Pull request merged**